product-mini/platforms/linux-sgx/enclave-sample/App/App.cpp

```diff
@@ -232,6 +232,7 @@ print_help()
     printf("                         for example:\n");
     printf("                           --addr-pool=1.2.3.4/15,2.3.4.5/16\n");
     printf("  --max-threads=n        Set maximum thread number per cluster, default is 4\n");
+    printf("  --version              Get the WAMR version number\n");
```

Comment:
How about changing to `Show version information` or `Print version information` like other runtimes?